content/updates/2020-05/2-2-way-associations.md

```diff
@@ -0,0 +1,19 @@
+# Product associations update
+
+## New association type: the 2-way association
+::: meta-data type="New" features="Product associations, Productivity, Data Governance" available="mid-May" link-to-doc="../articles/manage-your-association-types.html"
+:::
+
+We created a new association type named **2-way association**, accessible from `Settings` then `Association types` tabs.
+
+When associating some products to others, you now have the possibility to link them via a two-way association.
+Example: if you associate product A to product B, product B will be **automatically** associated to product A.
+
+This feature is especially useful for clients who need to display which products are compatible with which other products (like machines and spare parts).
+
+![2-way association type](../img/Associations_2-way-association-type.png)
+
+::: more
+[What are association types?](../articles/manage-your-association-types.html) or 
```

Comment:
```suggestion
[What are association types?](../articles/manage-your-association-types.html)  
```

Reply:
I added "or" because otherwise it displays the articles next to each others, and we barely see they are 2 different articles. There's no line break :/

Reply:
There is a line break if you put two spaces at the end of the first line. :wink: 
It's Markdown classical behavior.

Reply:
Oh, thanks Nolwenn.